doc/code/qml_fermi.rst

```diff
@@ -0,0 +1,87 @@
+qml.fermi
+=========
+
+Overview
+--------
+
+This module contains functions and classes for creating and manipulating fermionic operators.
+
+.. currentmodule:: pennylane.fermi
+
+.. automodapi:: pennylane.fermi
+    :no-heading:
+    :no-main-docstr:
+    :no-inherited-members:
+
+FermiC and FermiA
+-----------------
+
+The fermionic creation and annihilation operators are constructed with :class:`~pennylane.FermiC`
+and :class:`~pennylane.FermiA`, respectively, by passing the index of the orbital that the fermionic
+operator acts on. For instance, the operators :math:`a^{\dagger}_0` :math:`a_3` are constructed as
+
+>>> qml.FermiC(0)
+>>> qml.FermiA(3)
+
+These operators can be multiplied by each other to create :math:`n`-orbital fermionic operators such
+as :math:`a^{\dagger}_0 a_0 a^{\dagger}_3 a_3` that we call a Fermi word.
+
+>>> qml.FermiC(0) * qml.FermiA(0) * qml.FermiC(3) * qml.FermiA(3)
+<FermiWord = '0+ 0- 3+ 3-'>
+
+The Fermi words can be linearly combined to create fermionic operators that we call a Fermi
+Sentence. For instance, a fermionic Hamiltonian such as
+:math:`H = 1.2 a^{\dagger}_0 a_0  + 2.3 a^{\dagger}_3 a_3` can be constructed as
+
+>>> h = 1.2 * qml.FermiC(0) * qml.FermiA(0) + 2.3 * qml.FermiC(3) * qml.FermiA(3)
+>>> h
+1.2 * '0+ 0-'
++ 2.3 * '3+ 3-'
+
+These fermionic objects can be mapped to the qubit basis by using the function
+:func:`jordan_wigner`
+
+>>> qml.jordan_wigner(h)
+((-1.75+0j)*(Identity(wires=[0]))) + ((0.6+0j)*(PauliZ(wires=[0]))) + ((1.15+0j)*(PauliZ(wires=[3])))
+
+FermiWord and FermiSentence
+---------------------------
+
+Fermi words and Fermi Sentences can also be constructed directly with
+:class:`~pennylane.fermi.FermiWord` and :class:`~pennylane.fermi.FermiSentence` by passing
+dictionaries that define the fermionic operators.
+
+For Fermi words, the dictionary items identify the fermionic creation and annihilation operators.
```

Comment:
```suggestion
For Fermi words, the dictionary items define fermionic creation and annihilation operators.
```

I'm not sure this is the best way to phrase it either, but I was confused by 'identify'